config/rbac/kubernetes/kustomization.yaml

```diff
@@ -15,22 +15,26 @@
 # limitations under the License.
 # ---------------------------------------------------------------------------
 
+#
+# rbac resources applicable for all kubernetes platforms
+#
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
 resources:
+- user-global-kamelet-viewer-role.yaml
```

Comment:
If that's the case then they should be moved to infrastructure. I can check that out.

Reply:
Yes, I'd propose to move these resources out of the `config` directory. I find it confusing to mix operator resources and operand resources. I'd propose to move them into the `deploy` directory, and rename this directory to `resources`, to avoid confusing people that this contains deployment resources.